explorer/nginx/conf.d/nginx.conf.envsubst

```diff
@@ -84,13 +84,13 @@ http {
             include conf.d/enable-cors.conf;
         }
         # map selected prometheus queries to concrete endpoints that are publicly available
-        location = /explorer/shards {
+        location = /explorer/stored-per-shard {
             auth_basic off;
             limit_except GET {
                 deny all;
             }
             # https://www.nadeau.tv/nginx-proxy_pass-dns-cache/
-            set ${DOLLAR}backend "http://prometheus:9090/prometheus/api/v1/query?query=radixdlt_core_shards%7Bkey%3D~%22low%7Chigh%22%7D";
+            set ${DOLLAR}backend "http://prometheus:9090/prometheus/api/v1/query?query=sum%28radixdlt_core_ledger%7Bjob%3D%22nodes%22%2Ckey%3D%22stored_per_shard%22%7D%29%2A%282%5E44%2F%28sum%28radixdlt_core_shards%7Bkey%3D%22high%22%7D%29-sum%28radixdlt_core_shards%7Bkey%3D%22low%22%7D%29%29%29";
```

Comment:
Maybe consider adding the query as a human readable comment as well (you'll thank me later ;-) )